apps/prairielearn/src/pages/instructorInstanceAdminPublishing/components/LegacyAccessRuleCard.tsx

```diff
@@ -0,0 +1,103 @@
+import { formatDate } from '@prairielearn/formatter';
+
+import { CommentPopover } from '../../../components/CommentPopover.js';
+import type { CourseInstance, CourseInstanceAccessRule } from '../../../lib/db-types.js';
+
+export function LegacyAccessRuleCard({
+  accessRules,
+  showComments,
+  courseInstance,
+  hasCourseInstancePermissionView,
+}: {
+  accessRules: CourseInstanceAccessRule[];
+  showComments: boolean;
+  courseInstance: CourseInstance;
+  hasCourseInstancePermissionView: boolean;
+}) {
+  return (
+    <div class="card mb-4">
+      <div class="card-header bg-primary text-white d-flex align-items-center justify-content-between">
+        <h1>{courseInstance.long_name} course instance access rules</h1>
```

Comment:
Nit, simplification as long as we're here (offers consistency with the "General" tab):

```suggestion
        <h1>Access rules</h1>
```